Controller/ConnectController.php

```diff
@@ -466,7 +466,7 @@ private function getConfirmationResponse(Request $request, array $accessToken, $
         if ($currentToken instanceof OAuthToken) {
             // Update user token with new details
             $newToken =
-                is_array($accessToken) &&
+                \is_array($accessToken) &&
```

Comment:
Why are you changing lines like this one for this PR?

Maker less changes will make the PR more likely to get merged faster.

You can extract the code style changes in another PR

Reply:
My thinking was it's a separate commit, so if the maintainers want me to drop it I'll just drop it and be done with it. 🤷‍♀️